myuw/dao/term.py

```diff
@@ -391,6 +403,10 @@ def add_term_data_to_context(request, context):
 
     context['first_day'] = cur_term.first_day_quarter
     context['last_day'] = cur_term.last_day_instruction
+    context["first_day_quarter"] = cur_term.first_day_quarter
+    context["last_day_instruction"] = cur_term.last_day_instruction
```

Comment:
I would say the week should start on Monday.  Though - if that date is the meaningful value for a student, the first day should be the first day of classes, regardless of the day of week.  So for Autumn quarter it should (normally) be Wednesday